app/entry.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+require('./scss/main.scss');
+
+const angular = require('angular');
+const uiRouter = require('angular-ui-router');
+const ngFileUpload = require('ng-file-upload');
+
+angular.module('demoApp', [uiRouter, ngFileUpload])
+.config(['$stateProvider', '$urlRouterProvider',  function($stateProvider, $urlRouterProvider){
+  $urlRouterProvider.when('', '/');
+  let routes = [
+    {
+      name: 'home',
+      url: '/',
+      template: './component/landing/index.html',
+    },
+    {
+      name: 'gallery',
+      url: '/gallery',
+      template: './component/gallery-container/index.html',
+    },
+  ];
+
+  routes.forEach(route => $stateProvider.state(route));
+}]);
+
+// require services
+require('./service/auth-service.js');
+require('./service/photo-service.js');
```

Comment:
You don't have a ``photo-service.js`` yet, so this breaks your build and I'm unable to view anything you've done on ``localhost:8080``. Either include a photo-service or take this out for now so I can review without errors.